bin/configs/java-microprofile-server-multiple-2xx-responses.yaml

```diff
@@ -0,0 +1,8 @@
+generatorName: microprofile
+outputDir: samples/server/petstore/microprofile
```

Comment:
please add the new folder to `.github/workflows/samples-java-server-jdk8.yaml` so that CI will test it moving forward

Reply:
done